ElectronClient/app/gui/ConfigScreen.jsx

```diff
@@ -109,6 +109,19 @@ class ConfigScreenComponent extends React.Component {
 					<input type="text" style={controlStyle} value={this.state.settings[key]} onChange={(event) => {onTextChange(event)}} />
 				</div>
 			);
+		} else if (md.type === Setting.TYPE_INT) {
+			const onNumChange = (event) => {
+				const settings = Object.assign({}, this.state.settings);
+				settings[key] = event.target.value;
+				this.setState({ settings: settings});
+			};
+
+			return (
+        <div key={key} style={rowStyle}>
+          <div style={labelStyle}><label>{md.label()}</label></div>
+          <input type="number" style={controlStyle} value={this.state.settings[key]} onChange={(event) => {onNumChange(event)}} min={md.minimum} max={md.maximum} step={md.step}/>
+        </div>
+			);
```

Comment:
For some reason this part still has spaces instead of tabs so indentation is not right. Other than that it looks good.

Reply:
Should be solved now.